cdc/log.cc

```diff
@@ -63,6 +63,7 @@ static schema_ptr create_log_schema(const schema&, std::optional<utils::UUID> =
 }
 
 static constexpr auto cdc_group_name = "cdc";
+static const sstring cdc_partitioner_name = "com.scylladb.dht.CDCPartitioner";
```

Comment:
Could we perhaps make this string an extern instead of a duplicate static in two files?

Reply:
Yes, though because of the linkage of cdc_partitioner.o it is more than just remove static and add extern, but it is the nicer approach.

Reply:
Fixed.